lib/modules/apostrophe-pieces-orderings-pieces/lib/cursor.js

```diff
@@ -42,22 +42,24 @@ module.exports = {
               } else {
                 return body(null);
               }
-              function body(err) {
```

Comment:
This triggered: "Move function declaration to function body root  `no-inner-declarations`"

Reply:
It was already there? Weird